build.gradle

```diff
@@ -40,24 +40,21 @@ ext {
     supportAnnotations = 'com.android.support:support-annotations:21.0.3'
     rxJava = 'io.reactivex:rxjava:1.0.8'
 
-    junit = 'junit:junit:4.12'
-    mockitoCore = 'org.mockito:mockito-core:1.10.19'
-    testingSupportLib = 'com.android.support.test:testing-support-lib:0.1'
-
-
-    // Dagger 2 distributed as SNAPSHOT, we don't want to allow it break our builds, so we have local jars
-    dagger = 'com.google.dagger:dagger:2.0-SNAPSHOT' //files('libs/dagger-2.0-20150309.210909-16.jar')
-    daggerCompiler = 'com.google.dagger:dagger-compiler:2.0-SNAPSHOT' //files('libs/dagger-compiler-2.0-20150309.210931-16-jar-with-dependencies.jar')
-    daggerProducers = files('libs/dagger-producers-2.0-20150309.210913-4.jar')
```

Comment:
Yep, when I tried to use downloaded jars I needed it, but when we use it as maven dependency it resolved via Gradle with no problems, it's part of `daggerCompiler` dependencies.

As you can see, build is not failing because `project.ext.daggerProducers` even not used by any of our Gradle submodules